### PR TITLE
Create 7 Day Writing Challenge.md

### PR DESCRIPTION
**Pull request by smainalli**

### CODE REVIEW

**Review by samuel0kello — APPROVED** [review]

File: 7 Day Writing Challenge/Rules.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# ✍️ 7 Day Writing Challenge
```
Comment (codewithvoid) [inline 7 Day Writing Challenge/Rules.md]: rename this to announcement.md

File: 7 Day Writing Challenge/Step_1.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+## 🔶 Step1.Topic/Idea
```
Comment (codewithvoid) [inline 7 Day Writing Challenge/Step_1.md]: rename this file to day 1.md

File: 7 Day Writing Challenge/Step_1.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+The challenge will begin the day after you sign up.
+And pick a topic/title (pick up a domain and stick to it).
+
+### Brainstorm Ideas
+> One of the best ways to generate ideas of any kind is brainstorming. The more ideas you can come up with, the great your chances are of finding a good one.
+
+You can brainstorm in a bunch of different ways, including as a group, but here's the approach to it that works best for me:
```
Comment (codewithvoid) [inline 7 Day Writing Challenge/Step_1.md]: format this better.

File: 7 Day Writing Challenge/Step_2.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+## 🔶 Step2.Research
+Brainstorm ideas or look up on web for more information and resources.
+
+## Conduct thorough research about the topic
+While planning out your document, you have to research the topic you're writing about. There are tons of resources only a Google search away for you to consume and get deeper insights from.
+
+Don't be tempted to lift off other people's works or articles and pass it off as your own, as this is plagiarism. Rather, use these resources as references and ideas for your work.
+
+Google as much as possible, get facts and figures from research journals, books or news, and gather as much information as you can about your topic. Then you can start making an outline.
```
Comment (codewithvoid) [inline 7 Day Writing Challenge/Step_2.md]: - Shorten the steps
- Put it in a form of 1.2.3
- Give examples wherever necessary

**Review by codewithvoid — CHANGES_REQUESTED** [review]

File: 7 Day Writing Challenge/announcement.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+**🔶 Step3.Drafting**: Write the first draft and have a intro, middle and end.
+
+**🔶 Step4.Editing**:Use tools to check up on grammar and any corrections. At the end of each day, you’ll let us know how did you wrote that day.
+
+**🔶 Step5.Review**: At this stage, the article should be around 75% complete. Well Done! The first draft is complete.We will pick your blog for review and feedback will be shared.
+
+**🔶 Step6.Publish**: Now that you have the idea, the goal, the structure, and some feedback, it’s time to polish everything up and ship it.
+
+**🔶 Step7.Share**: Once you’ve actually published the post, you’re not done! If you want people to see what you’ve written and get value out of it, make sure to share it in the places where your audience is likely to hang out.
```
Comment (codewithvoid) [inline 7 Day Writing Challenge/announcement.md]: publish and share can be together

File: 7 Day Writing Challenge/Day_1.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
+## 🔶 Day 1. Topic/Idea
```
Comment (codewithvoid) [inline 7 Day Writing Challenge/Day_1.md]: rename folder name to 7_DAY_CHALLENGE

File: 7 Day Writing Challenge/announcement.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+**🔶 Step5.Review**: At this stage, the article should be around 75% complete. Well Done! The first draft is complete.We will pick your blog for review and feedback will be shared.
+
+**🔶 Step6.Publish**: Now that you have the idea, the goal, the structure, and some feedback, it’s time to polish everything up and ship it.
+
+**🔶 Step7.Share**: Once you’ve actually published the post, you’re not done! If you want people to see what you’ve written and get value out of it, make sure to share it in the places where your audience is likely to hang out.  
+
+> At the end of the 7 days, you’ll have written your first blogpost! More importantly, you have begun your writing habit!
+### 🤘  Congratulate Yourself
+Within a week, you undeniably learnt a thing or two, to add with that, you wrote an article, so congratulate yourself.
```
Comment (codewithvoid) [inline 7 Day Writing Challenge/announcement.md]: add new line here

File: 7 Day Writing Challenge/announcement.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+## 🤚 How to get help
+When you have a question, hit up the `#7day-writing-challenge` [channel](https://discord.com/channels/1073966100552896632/1073983804437450763) .
```
Comment (codewithvoid) [inline 7 Day Writing Challenge/announcement.md]: update channel name to writing-discuss

File: 7 Day Writing Challenge/announcement.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+
+## 📜 Conclusion 
+Well, I guess the challenge has been explained enough. If you need more clarifications, kindly reach me on [twitter]() by using #Askdevs
```
Comment (codewithvoid) [inline 7 Day Writing Challenge/announcement.md]: `I guess the challenge has been explained enough.` remove this line

Comment (codewithvoid) [inline 7 Day Writing Challenge/announcement.md]: add askdevs intent link

File: 7 Day Writing Challenge/announcement.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+When you have a question, hit up the `#7day-writing-challenge` [channel](https://discord.com/channels/1073966100552896632/1073983804437450763) .
+
+## 📜 Conclusion 
+Well, I guess the challenge has been explained enough. If you need more clarifications, kindly reach me on [twitter]() by using #Askdevs 
+
+`#7day-writing-challenge` is officially launched 🚀
+
+Commit to the Challenge Now!!
+
+Let's get you some open-source rep!!!
```
Comment (codewithvoid) [inline 7 Day Writing Challenge/announcement.md]: nit: Let's get your first blogpost published.

File: 7 Day Writing Challenge/announcement.md [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+
+At the very least, drop a ⭐ on the repo if you're feeling fancy!
```
Comment (codewithvoid) [inline 7 Day Writing Challenge/announcement.md]: Leave a star to support us 🙏

File: 7 Day Writing Challenge/announcement.md [hunk 12]
```diff
@@ -0,0 +1,47 @@
...
+Within a week, you undeniably learnt a thing or two, to add with that, you wrote an article, so congratulate yourself.
+## 🤚 How to get help
+When you have a question, hit up the `#7day-writing-challenge` [channel](https://discord.com/channels/1073966100552896632/1073983804437450763) .
+
+## 📜 Conclusion 
+Well, I guess the challenge has been explained enough. If you need more clarifications, kindly reach me on [twitter]() by using #Askdevs 
+
+`#7day-writing-challenge` is officially launched 🚀
+
+Commit to the Challenge Now!!
```
Comment (codewithvoid) [inline 7 Day Writing Challenge/announcement.md]: Join the community and commit to the challenge now!

Add discord link.

**Comment by justshubhi** [comment]
closing this.

**Pull request closed without merging**